src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs

```diff
@@ -65,13 +76,24 @@ public void DocumentTransformerAsDelegate_Delegate()
         _documentService = CreateDocumentService(_builder, _options);
     }
 
-    [GlobalSetup(Target = nameof(SchemaTransformer))]
+    [GlobalSetup(Target = nameof(ActivatedSchemaTransformer))]
+    public void ActivatedSchemaTransformer_Setup()
+    {
+        _builder.MapGet("/", () => { });
```

Comment:
We should provide a `Todo` parameter here so the schema transformer paths actually get executed.